### PR TITLE
allow to search for user id in admin/index view

### PR DESCRIPTION
**Pull request by thyseus**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no

### CODE REVIEW

File: models/UserSearch.php [hunk 1]
```diff
@@ -95,6 +99,7 @@ public function search($params)
 
         $query->andFilterWhere(['like', 'username', $this->username])
             ->andFilterWhere(['like', 'email', $this->email])
+            ->andFilterWhere(['user.id' => $this->id])
```
Comment (thiagotalma) [inline models/UserSearch.php]: You should not use it like that.
The model may have been overwritten and be using another table name.

Comment (philippfrenzel) [inline models/UserSearch.php]: Hi, pls. even replace user.id with {{%user}}.id so if a table prefix is set, it will be applied

Comment (thiagotalma) [inline models/UserSearch.php]: In fact, the correct one would be to use `$query->modelClass::tableName() . '.id'`

We should always remember that models can be remapped.

Please make the change and I will accept the PR.

**Comment by thyseus** [comment]
Why do we need to do the tableName alias? Because the UserSearch will (probably) get a JOIN in the future: https://github.com/dektrium/yii2-user/pull/835

**Comment by thiagotalma** [comment]
Thanks!

**Pull request merged**